style.css

```diff
@@ -0,0 +1,633 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap');
+
+html,
+body {
+  overflow-x: hidden;
+}
+
+body {
+  font-family: "Lato", sans-serif;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.main-page {
+  background-color: #fff;
+  margin: 0;
+}
+
+header {
+  display: flex;
+  margin: 0 30px;
+  gap: 40%;
+}
+
+.top {
+  display: none;
+  list-style: none;
+  gap: 20px;
+  justify-content: flex-end;
+  padding: 10px 20px;
+}
+
+.lg-logo {
+  display: none;
+}
+
+.logo {
+  padding: 10px;
+  font-weight: 700;
+  display: none;
+}
+
+.nav-1 {
+  display: flex;
+}
+
+.list {
+  gap: 30%;
+  list-style: none;
+  font-weight: 300;
+  color: #272a31;
+  padding: 10px;
+  display: none;
+}
+
+.list-li {
+  color: #ec5242;
+  list-style: none;
+}
+
+.menu {
+  cursor: pointer;
+  align-self: center;
+}
+
+.bar {
+  display: block;
+
+  /* width: 35px; */
+  margin: 4px auto;
+  background-color: black;
+  height: 2px;
+}
+
+.show-links {
+  height: 100%;
+  width: 100%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  background: #6c757d;
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  padding-top: 130px;
+  transform: translate(0, -20px);
+  gap: 30px;
+}
+
+.show-links a {
+  color: #fff;
+  display: flex;
+  font-family: 'cocogoose', sans-serif;
```

Comment:
- [ ] You did great using the `Cocogoose` font family as requested by the previous reviewer but make sure that you have already imported it because normally these special google fonts must be imported as you imported `Lato`. Please fix that. :wink: